README.md

```diff
@@ -80,9 +77,9 @@ Additionally to the information provided on your application, note that your pro
     2. Fully or partially public applications should be completed via a pull request to this repo by following the steps below.
        1. [Fork](https://github.com/w3f/General-Grants-Program/fork) this repository.
        2. In the newly created fork, create a **copy** of the application template ([`grants/grant_application_template.md`](./grants/grant_application_template.md)). If you're using the GitHub web interface, make sure you create a new file and copy the content of the template inside the new file. _Don't change the template file directly!_
-           - In the few cases where the application is a response to a specific RFP (Request For Proposal), the application goes into the [`rfps`](./rfps) folder.
+           - In the few cases where the application is in response to a specific RFP (Request For Proposal), the application goes into the [`previous-rfps`](./grants/previous-rfps/) folder. In this case, you may also remove the RFP from the [`rfps`](./rfps) folder in your pull request.
```

Comment:
Really? I thought it was "Request for Proposal**s**", so more than one team can apply.

Reply:
I think you're right, I forgot. Although seeing how little interest we are currently getting for RFPs, it might be difficult to get more than one team to apply before we accept. Let's see what @Noc2 says

Reply:
The document now calls for multiple teams to apply.